scripts/install

```diff
@@ -130,16 +131,15 @@ ynh_replace_string "__USER__" "${app}" /etc/sudoers.d/${app}_ynh
 mkdir -p $final_path/db/server
 
 # Add interface configuration file for the Web UI
-cp ../conf/interfaces.json $final_path/db/server/interfaces.json
-ynh_replace_string --match_string="__PORT_WG__" --replace_string="$port_wg" --target_file="$final_path/db/server/interfaces.json"
+ynh_add_config --template="../conf/interfaces.json" --destination="$final_path/db/server/interfaces.json"
+ynh_delete_file_checksum --file="$final_path/db/server/interfaces.json"
```

Comment:
Uuuh why are we deleting the checksum tho :thinking:

Reply:
Anytime the app gets upgraded it would complain about the file being modified, while it's implied they will get altered by the user and the UI.

Reply:
Do you think there's a better way to handle that?